app/Livewire/Settings/Updates.php

```
@@ -0,0 +1,101 @@
+            }
+
+            $this->instantSave();
+            $this->server->setupDynamicProxyConfiguration();
+        } catch (\Exception $e) {
+            return handleError($e, $this);
+        }
+    }
+
+    public function checkManually()
+    {
+        CheckForUpdatesJob::dispatchSync();
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Jobs\CheckForUpdatesJob' in method 'checkManually'. (Clean Code Rules)

(StaticAccess)